src/Dofs/DofHandler.jl

```diff
@@ -312,15 +312,22 @@ function cellnodes!(global_nodes::Vector{Int}, grid::Grid{dim,C}, i::Int) where
     return global_nodes
 end
 
-function cellcoords!(global_coords::Vector{Vec{dim,T}}, grid::Grid{dim,C}, i::Int) where {dim,C,T}
-    nodes = grid.cells[i].nodes
+# shared implementation for all grids
+function cellcoords!(global_coords::Vector{Vec{dim,T}}, grid::AbstractGrid, cell::C) where {dim,C<:Ferrite.AbstractCell,T}
+    nodes = cell.nodes
     N = length(nodes)
     @assert length(global_coords) == N
     for j in 1:N
         global_coords[j] = grid.nodes[nodes[j]].x
```

Comment:
There is an interface function which should be used, because the functionality of `cellcoords!` can be recovered by the already defined interface functions. I think it would be worth considering to put this function in another file, e.g. in the grid.jl file, doesn't seem to be exclusively related the dofhandler
```suggestion
        global_coords[j] = getnodes(grid,nodes[j]).x
```

Reply:
Using the interface function is definitely the right way to go. 

I also agree that this should live in the grid file. I simply didn't move it yet so that it's easier to see the actual diff caused by this PR (and not the one by shuffling around functions).